api/src/main/java/io/strimzi/api/kafka/model/status/HttpListenerAddress.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Represents a single address of particular listener
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "host", "port", "url" })
+@EqualsAndHashCode(callSuper = true)
+public class HttpListenerAddress extends ListenerAddress {
+    private static final long serialVersionUID = 1L;
+
+    private String url;
```

Comment:
When I talked about adding a URL before I was speaking hypothetically: Do we really want to add it as part of this PR?